### ISSUE TITLE
JSON hash varies exporting with identical settings the same skeleton

### ISSUE THREAD

**Issue by erikari** [EsotericSoftware/spine-editor#503]
**Repro steps:**
1. Open mix-and-match project
2. Export a json (in my case I used: no pack, yes pretty print, no clean up, no nonessential data)
3. rename the exported json so it doesn't get overwritten
4. Export again with same settings
> Different hashes

**Reference topic:**
http://esotericsoftware.com/forum/JSON-export-is-random-since-3-8-12989

**Comment by smaren** [comment]
Hi, creator of the topic here. The issue for us is not the fact that the hashes changes, but rather that objects within the json are reordered. Example:

```
"animations": {
	"toggled": {
		"slots": {
			"label": {
				"color": [
					{ "color": "18ff00ff" }
				]
			},
			"button": {
				"color": [
					{ "color": "18ff00ff" }
				]
			}
		}
	}
}
```

is changed to

```
"animations": {
	"toggled": {
		"slots": {
			"button": {
				"color": [
					{ "color": "18ff00ff" }
				]
			},
			"label": {
				"color": [
					{ "color": "18ff00ff" }
				]
			}
		}
	}
}
```

Changing the order of the `button` and `label` slots.

**Comment by smaren** [comment]
I have tried exporting some of the example spines (owl, mix-and-match among others). Some produce different json with each export while others do not. The mix-and-match spine produces different json with each export, while the owl export seems to be identical.
Using spine version 3.8.75, command line export

```
{
"class": "export-json",
"name": "JSON",
"project": "/path/to/Spine/examples/mix-and-match/mix-and-match-pro.spine",
"output": "/path/to/export",
"open": false,
"extension": ".json",
"format": "JSON",
"prettyPrint": true,
"nonessential": false,
"cleanUp": false,
"packAtlas": null,
"packSource": "attachments",
"packTarget": "perskeleton",
"warnings": true
}
```

**Comment by NathanSweet** [comment]
Fixed in 3.8.76. Thanks!
